docs/build_version_doc/build_all_version.sh

```diff
@@ -21,7 +21,7 @@
 # Built files are stored in $built
 # Version numbers are stored in $tag_list.
 # Version numbers are ordered from latest to old and final one is master.
-tag_list="0.12.0 0.11.0 master"
+tag_list="1.0.0 0.12.0 0.11.0 master"
```

Comment:
1.0.0 is not available yet. 0.12.1 should be added instead.

Reply:
Let's hold off merging until we open branch

Reply:
Sure. 
@szha you wanted python/mxnet/libinfo.py updated with 1.0 correct?

Reply:
Yes. Currently we are still developing on a version number that already has a release, so it needs to be bumped up.

Reply:
@szha Could you please merge this into the 1.0.0 branch

Reply:
@mbaijal would you please update the tag with 0.12.1?

Reply:
@piiswrong after @mbaijal updates the tag list, this PR won't involve any change that needs the 1.0.0 tag. In addition, current nightly builds are still released as 0.12.1beta versions because this PR is blocked. Could we merge this PR after @mbaijal makes the update, to resolve the nightly build situation?

Reply:
@szha 
I have reverted the tag to 0.12.1 in the two places mentioned. Does this look ok now?
However I think this build script 'build_all_version.sh' is only for local testing it does not actually need the listed tag to be available on github.